lib/rubocop/cop/rspec/implicit_subject.rb

```diff
@@ -42,21 +42,36 @@ module RSpec
       #   # good
       #   it { expect(subject).to be_truthy }
       #
+      # @example `EnforcedStyle: require_implicit`
+      #   # bad
+      #   it { expect(subject).to be_truthy }
```

Comment:
Another thing that is unsettling about this bad example is that it is using an unnamed subject. I understand that other examples do, too.
I'm uncertain how to clarify that even if the subject is named, it will be enforced to be turned into `is_expected`.

Reply:
We have separate cop for dealing with wether subject is named or not. If the subject is named however, it would be forced to be `is_expected`